src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    public String getPrintableString (Printable... printables);
```

Comment:
You can omit the `public` modifier. All abstract methods in an interface is implicitly public.